docs/src/howto-guides/011_retrieve_jwt_token_from_cumulocity.md

```diff
@@ -0,0 +1,24 @@
+## How to retrieve the JWT Token
```

Comment:
The title must explicit the context

```suggestion
## How to retrieve a JWT Token to authenticate on Cumulocity
```

I would also add a short introduction.

```
In order to [authenticate HTTP requests on Cumulocity](https://cumulocity.com/guides/10.5.0/reference/rest-implementation/#authentication), a device can retrieve a JWT token using MQTT. 
```